transcoder/src/extract.go

```diff
@@ -8,26 +8,45 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"time"
 
 	"github.com/zoriya/kyoo/transcoder/src/storage"
 	"github.com/zoriya/kyoo/transcoder/src/utils"
 	"golang.org/x/sync/errgroup"
 )
 
-const ExtractVersion = 1
+const (
+	ExtractVersion = 1
+	maxJobDuration = 5 * time.Minute // TODO make this configurable
```

Comment:
I don't think we need to make this configurable. im leaning on `the less options we have, the better`

Reply:
Sure, I can do this. I personally usually err on the side of more config, but sane defaults (convention over configuration). But IMO it is unlikely that anybody will really need to tune this, and it can be revisited if somebody does want this.